tests/garage/experiment/test_resume.py

```diff
@@ -58,4 +58,5 @@ def test_resume(self):
 
             runner.resume(n_epochs=10)
 
+        snapshotter.reset()
```

Comment:
these two lines should be in teardown

Reply:
Yes... Let me make the changes and see if it breaks anything.

Reply:
Now `snapshotter.reset()` in `tearDown()`. Maybe not the create and delete of temp folder since it's only used in test_resume() now.